content/concepts/kata/translations.md

```diff
@@ -77,21 +81,39 @@ After a review is completed, the reviewer can finalize it with one of following
 
 ### Approving a translation
 
-TBD
+Pending translation can be approved either by kata author, or users with [privilege to coauthor](/references/gamification/privileges/) who are not the publisher of the translation fork being approved, under following conditions:
+
+- translation is older than a week, or
+- kata author has been inactive for over a month.
+
+When translation is approved, new language version is immediately available for other users to solve. There's also a honor reward, but remember that it's granted to the author of the **initial fork**, and not to the author of the fork which got finally accepted!
 
-- who can approve
-- Approver is marked as coauthor
+Translation approver should also go through the kata discourse and mark suggestions posted by the translator as resolved.
+
+**NOTE:** currently, probably because of a bug, a user who _approves_ a translation is marked as **kata contributor**. Users with "contributor" flag seem to be affected by a bug which prevents them from getting any future points they would be rewarded for authoring activities on the kata they contributed to. Be aware that as a result, approving of a translation can cause you won't be eligible for any Honor points you would earn by translating, editing or fixing the kata in the future, unless the bug is fixed.
 
 ### Merge Issues
 
-Oftentimes if two translations are created around the same time, they may both update the description. When approving the 2nd translation, the 1st approved translation will have already updated the description. This will cause a merge conflict which will need to be resolved. You can do this by forking the 2nd translation and updating the description. `<<<<<<<<`, `=========` and `>>>>>>>>` merge labels will show up in the description to help identify where the merge issues are.
+Oftentimes if two translations are created around the same time, they may both update the description. When approving the 2nd translation, the 1st approved translation will have already updated the description. This will cause a merge conflict which has to be resolved. You can do this by forking the 2nd translation and updating the description. `<<<<<<<<`, `=========` and `>>>>>>>>` merge labels will show up in the description to help identify where the merge issues are.
 
 Once you have fixed the description merge issues you can publish the translation. If you are not the author then you will need to have someone else verify your changes and approve the translation.
 
 ### Rejecting a translation
 
+Sometimes the best course of action is to simply reject a pending translation. Some possible reasons could be:
+
+- translation is outdated and kata changed a lot since the translation has been published,
+- translation is of bad quality and fixing it is too much work,
+- some user authored newer, better translation and they cannot publish it because there can be only one pending translation per kata per language,
```

Comment:
```suggestion
- some user authored a newer, better translation and they cannot publish it since there can only be one pending translation per language per kata,
```